src/test/java/friday/storage/JsonFridayStorageTest.java

```diff
@@ -16,8 +16,8 @@
 import friday.model.ReadOnlyFriday;
 import friday.testutil.TypicalStudents;
 
-public class JsonMasteryCheckBookStorageTest {
-    private static final Path TEST_DATA_FOLDER = Paths.get("src", "test", "data", "JsonMasteryCheckBookStorageTest");
+public class JsonFridayStorageTest {
+    private static final Path TEST_DATA_FOLDER = Paths.get("src", "test", "data", "JsonFridayStorageTest");
```

Comment:
Good to see that you have rectified this. This file was quite deep in the file system and yo managed to catch it good job.